test/core/pipeline/test_draw.py

```diff
@@ -142,20 +143,6 @@ def test_to_mermaid_image_invalid_format():
         _to_mermaid_image(pipe.graph, params={"format": "invalid_format"})
 
 
-@pytest.mark.integration
-def test_to_mermaid_image_missing_theme():
```

Comment:
this is almost a duplicate of [`test_to_mermaid_image`](https://github.com/deepset-ai/haystack/blob/4773d576bcb6181abd02794aa11dda27cef2114f/test/core/pipeline/test_draw.py#L17), so I would remove it.

Reply:
good point